### DISCUSSION TITLE
Herokuapp trouble

### DISCUSSION THREAD

**Discussion by Chuchi12345** [anirudhmalik/xhunter#86] [General]
So ive done everything that the instructions say and I was able to deploy my herokuapp. My problem is that when I choose to listen on the xhunter app using the heroku server it stays connecting forever and never actually connects. Any suggestions?

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
